src/Entity/Adherent.php

```diff
@@ -2195,45 +2256,73 @@ public function isLre(): bool
         return $this->lreArea instanceof LreArea;
     }
 
-    public function getAllReferentManagedTagsCodes()
+    public function getFacebookPageUrl(): ?string
     {
-        $tags = [];
-        if ($this->isReferent()) {
-            $tags = $this->getManagedArea()->getReferentTagCodes();
-        }
+        return $this->facebookPageUrl;
+    }
 
-        foreach ($this->getReceivedDelegatedAccessOfType('referent') as $delegatedAccess) {
-            $tags = \array_merge($tags, $delegatedAccess->getDelegator()->getManagedArea()->getReferentTagCodes());
-        }
+    public function setFacebookPageUrl($facebookPageUrl): void
```

Comment:
It should be possible to set `null` to `facebookPageUrl`.
Same for other urls' setters.